cli/cmd/errors_lql.go

```diff
@@ -0,0 +1,101 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/lacework/go-sdk/internal/failon"
+)
+
+type queryPolicyError struct {
```

Comment:
nit.  is there a better term other than `queryPolicy`.  Given the association of queries with LPP (platform policies) this is slightly misleading.

Reply:
the important part is the `<type>Error` - Any name you want this to be called? I was thinking to remove the internal package you created `failon` and use the name of the struct defined there `CountOperationError`. Thoughts?